src/playermat/Playmat.ttslua

```diff
@@ -245,32 +245,26 @@ function doUpkeep(_, clickedByColor, isRightClick)
 
   -- unexhaust cards in play zone, flip action tokens and find forcedLearning
   local forcedLearning = false
+  local rot = self.getRotation()
   for _, v in ipairs(searchAroundSelf()) do
     local obj = v.hit_object
     if obj.getDescription() == "Action Token" and obj.is_face_down then
       obj.flip()
-    elseif obj.tag == "Card" and not obj.is_face_down and not inArea(self.positionToLocal(obj.getPosition()), INVESTIGATOR_AREA) then
+    elseif obj.tag == "Card" and not inArea(self.positionToLocal(obj.getPosition()), INVESTIGATOR_AREA) then
       local cardMetadata = JSON.decode(obj.getGMNotes()) or {}
       if not doNotReady(obj) then
-        obj.setRotation(self.getRotation())
+        -- set correct rotation for face-down cards
```

Comment:
This does work, but I have no idea why it works.  Generally, comments should explain why you're doing something rather than just what, since that can be seen from the code itself.  In this case, what changes about the coordinate space that makes the z-axis invert?

Reply:
Since the `setRotation()` needs a full vector as input, the z-value needs to be set according to the cards orientation. I could have also used `getRotation()` on each card and manipulated the x- and y-value, but the would be more steps.

Reply:
My point was that should go in the comment :)

Probably not worth changing now, but even if it's a little more code it's generally good practice not to mutate data unless you're actually intending to change it.  "Preserving the current Z value" is both clearer and safer than "Assume we know what the Z value should be."  I don't see any harm in this case though.